### PR TITLE
Add additional benchmarks

### PR DESCRIPTION
**Pull request by clncy**
- IO operations (to and from WKT and WKB)
- Geometry construction

Addresses issue #109.

### CODE REVIEW

File: benchmarks/benchmarks.py [hunk 1]
```diff
@@ -16,3 +16,41 @@ def time_distance(self):
 
     def time_intersection(self):
         pygeos.intersection(self.points, self.polygon)
+
+
+class IOSuite:
+    """Benchmarks I/O operations (WKT and WKB) on a set of 100000 polygons"""
+    def setup(self):
+        self.to_write = pygeos.polygons(np.random.random((100000, 2)))
```
Comment (jorisvandenbossche) [inline benchmarks/benchmarks.py:24]: Note that this is not actually 10000 polygons, but a single polygon of 10000 vertices. I think it is better to actually construct 10000 polygons (``pygeos.polygons(np.random.random((10000, 3, 2)))`` to ensure we benchmark pygeos' overhead, and not just the GEOS operation

**Comment by jorisvandenbossche** [comment]
Can you also how the output of `asv run` with those?

File: benchmarks/benchmarks.py (hunk 1 above)
Comment (clncy) [reply benchmarks/benchmarks.py:24]: Whoops. I'll fix that up.

**Comment by clncy** [comment]
@jorisvandenbossche Where do you want me to put the output?

**Comment by jorisvandenbossche** [comment]
Just here as we comment is fine

**Comment by caspervdw** [comment]
> Where do you want me to put the output?

Thanks @clncy !

If you make a gh-pages branch output the results in there, we can see it on https://clncy.github.io/pygeos/ .     For reference: https://pygeos.github.io/pygeos/

**Comment by jorisvandenbossche** [comment]
I don't think doing the effort of putting up a site is really needed for adding a few benchmarks as a PR?  

I mainly asked for the output of `asv run` just as a sanity check that the timings are in good order of magnitude. 

So I meant something like this, what I get for the current benchmarks (but then for the new ones):

```
$ asv run --python=same
· Discovering benchmarks
· Running 3 total benchmarks (1 commits * 1 environments * 3 benchmarks)
[  0.00%] ·· Benchmarking existing-py_home_joris_miniconda3_envs_geo-dev_bin_python
[ 16.67%] ··· Running (benchmarks.PointPolygonTimeSuite.time_contains--)...
[ 66.67%] ··· benchmarks.PointPolygonTimeSuite.time_contains                 1.39±0.01ms
[ 83.33%] ··· benchmarks.PointPolygonTimeSuite.time_distance                    57.0±2ms
[100.00%] ··· benchmarks.PointPolygonTimeSuite.time_intersection                377±50ms
```

**Comment by clncy** [comment]
I've fixed the issue now, thanks for your help.

The output of the benchmarks on my (not particularly powerful) laptop was:
```
% asv run
· Creating environments
· Discovering benchmarks
· Running 11 total benchmarks (1 commits * 1 environments * 11 benchmarks)
[  0.00%] · For pygeos commit 17854054 <master>:
[  0.00%] ·· Benchmarking conda-py3.7-geos-numpy
[  4.55%] ··· Running (benchmarks.ConstructiveSuite.time_convex_hull--).......
[ 36.36%] ··· Running (benchmarks.IOSuite.time_write_to_wkt--)....
[ 54.55%] ··· benchmarks.ConstructiveSuite.time_convex_hull                      5.32±0.1ms
[ 59.09%] ··· benchmarks.ConstructiveSuite.time_delaunay_triangles                84.7±10ms
[ 63.64%] ··· benchmarks.ConstructiveSuite.time_envelope                         3.70±0.2ms
[ 68.18%] ··· benchmarks.ConstructiveSuite.time_voronoi_polygons                   111±10ms
[ 72.73%] ··· benchmarks.IOSuite.time_read_from_wkb                                99.6±5ms
[ 77.27%] ··· benchmarks.IOSuite.time_read_from_wkt                                 415±6ms
[ 81.82%] ··· benchmarks.IOSuite.time_write_to_wkb                                 102±10ms
[ 86.36%] ··· benchmarks.IOSuite.time_write_to_wkt                               1.42±0.01s
[ 90.91%] ··· benchmarks.PointPolygonTimeSuite.time_contains                    3.08±0.08ms
[ 95.45%] ··· benchmarks.PointPolygonTimeSuite.time_distance                        226±8ms
[100.00%] ··· benchmarks.PointPolygonTimeSuite.time_intersection                  1.87±0.4s
```

**Comment by jorisvandenbossche** [comment]
OK, looks good to me, thanks!

**Pull request merged**